components/builder-worker/src/runner/logger.rs

```diff
@@ -13,26 +13,60 @@
 // limitations under the License.
 
 use std::fs::File;
-use std::io::Write;
+use std::io::{BufRead, BufReader, Write};
+use std::process;
 
 use super::workspace::Workspace;
 
+const EOL_MARKER: &'static str = "\n";
```

Comment:
Is this going to cause problems on Windows?  Once upon a time, Windows used "\r\n" as the line terminator.  Is that still a thing?  My windows knowledge is admittedly antiquated.